app/src/main/java/com/plangrid/plansensor/MainActivity.kt

```diff
@@ -14,6 +34,43 @@ class MainActivity : AppCompatActivity() {
         graphView.setData(listOf(1, 5, 6, 4, 2, 1, 1, 3, 8))
 
         // TODO(1): Listen to the sensor and display a running list of the most recent ten values in the graph.
+        val record = findViewById<Button>(R.id.record)
+        record.setOnClickListener {
+            // TODO: Implement recording
+        }
+        val play = findViewById<Button>(R.id.play)
+        play.setOnClickListener {
+            // TODO: Implement playback
+        }
+
+        // TODO(1): Listen to the sensor and display the last ten values in the graph.
```

Comment:
I think we also want to remove these `TODO(1)`s since they were confusing in the last interview we gave (having already been done in the part 2 branch). But that should be a separate PR